src/pymor/models/iosys.py

```diff
@@ -707,6 +743,8 @@ def hinf_norm(self, mu=None, return_fpeak=False, ab13dd_equilibrate=False):
         fpeak
             Frequency at which the maximum is achieved (if `return_fpeak` is `True`).
         """
+        if "hinf_norm" in self.presets and mu is None:
+            return self.presets["hinf_norm"]
```

Comment:
It would be nice to also have something like `'fpeak'`. If it is set, the function could simply return the norm of `self.transfer_function(presets['fpeak'])`.